frontend/src/lib/components/MapCenterControl.ts

```diff
@@ -10,31 +10,39 @@ export class MapCenterControl extends L.Control {
 		img.src = locationSearching;
 		img.style.cursor = 'pointer';
 		img.style.width = '25';
-		img.setAttribute('title', 'Center Map');
-
-		const div = L.DomUtil.create('div');
-		div.addEventListener('click', () => map.locate());
-		div.appendChild(img);
-		div.style.backgroundColor = '#fff';
-		div.style.border = '2px solid rgba(0, 0, 0, 0.2)';
-		div.style.backgroundClip = 'padding-box';
-		div.style.borderRadius = '4px';
-		div.style.width = '34px';
-		div.style.height = '34px';
-		div.style.display = 'flex';
-		div.style.justifyContent = 'center';
-		div.style.alignItems = 'center';
-
-		div.addEventListener('mouseenter', () => {
-			console.log('enter');
-			div.style.backgroundColor = '#f4f4f4';
-		});
-
-		div.addEventListener('mouseleave', () => {
-			div.style.backgroundColor = '#fff';
-		});
-
-		return div;
+
+		const control = L.DomUtil.create('a');
+		control.addEventListener('click', () => map.locate());
+		control.appendChild(img);
+		control.setAttribute('role', 'button');
+		control.setAttribute('title', 'Center Map');
+		control.setAttribute('aria-label', 'Center Map');
+		control.href = '#';
+		control.style.backgroundColor = '#fff';
```

Comment:
I wonder why all this styling was done in JavaScript instead of in a CSS file. It probably should be moved to CSS.